ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/SessionBeanComponentDescriptionFactory.java

```diff
@@ -174,6 +174,12 @@ private void processSessionBeans(final DeploymentUnit deploymentUnit, final List
             }
 
             addComponent(deploymentUnit, sessionBeanDescription);
+
+            final AnnotationValue mappedNameValue = sessionBeanAnnotation.value("mappedName");
+            if (mappedNameValue != null && !mappedNameValue.asString().isEmpty()) {
+                EjbLogger.ROOT_LOGGER.mappedNameNotSupported(mappedNameValue != null ? mappedNameValue.asString() : "",
+                        ejbName);
+            }
```

Comment:
`mappedNameValue != null ? mappedNameValue.asString() : "",` seems to be redundant, as mappedNameValue is already not null.

Reply:
consider the above comment as optional minor improvement, not blocking one. I've approved this PR.